keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlDeliveryConfigRepository.kt

```diff
@@ -166,31 +188,28 @@ class SqlDeliveryConfigRepository(
         .fetchOne(DELIVERY_CONFIG.UID)
     } ?: throw NoDeliveryConfigForApplication(application)
 
-  private fun getEnvironmentUIDs(deliveryConfigUIDs: List<String>): List<String> =
+  private fun getResourceUIDs(application: String): SelectConditionStep<Record1<String>>? =
     sqlRetry.withRetry(READ) {
       jooq
-        .select(ENVIRONMENT.UID)
```

Comment:
You also don't need to call this on a `DSLContext` instance, there's a static function `DSL.select` you can use when you want to create a sub-query like this.

Reply:
Yeah, but it doesn't hurt, does it?